how-it-works/00_General/01_overview-of-the-internet-computer.subpage.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Architecture of the Internet Computer
+abstract: 
+shareImage: /img/how-it-works/overview-of-the-internet-computer.600.jpg
+slug: architecture-of-the-internet-computer
+---
+
+# Architecture of the Internet Computer
+
+The technology behind the Internet Computer (IC) has been reconceived from ground up, learning lessons from the weaknesses of earlier projects, e.g., the lack of scalability.
+The Internet Computer intends to become the World Computer, an open and secure blockchain-based network that can scale out to power humanity's computation.
+
+## Subnet Architecture
```

Comment:
As above, I think it would be better to start with canisters and visit subnets as a scaling mechanism later. The fact that you need to forward-reference canisters in this paragraph seems to indicate that, too.